dbmongo/lib/engine/main_test.go

```diff
@@ -1,7 +1,6 @@
 package engine
 
 import (
-	"dbmongo/lib/engine"
```

Comment:
cette ligne faisait retourner un code d'erreur `1` à la commande `go get -v -t -d ./...` depuis la CI.

en exécutant la commande depuis un container docker, j'obtenais ce message d'erreur: `unrecognized import path "dbmongo/lib/engine": import path does not begin with`

Reply:
tout à fait, cela venait probablement du fait qu'il était écrit engine.TrackerReports plus bas, l'éditeur fabrique les lignes d'import à la demande sans forcément se demander si la fonction est déjà présente dans le namespace principal.